interface.go

```diff
@@ -27,6 +31,7 @@ type Database interface {
 	NewBatch() Batch
 	Path() string
 	NewIterator() Iterator
+	Subscribe(ctx context.Context, cb func(kv *badger.KVList) error, prefixes []byte) error
```

Comment:
is there a way to make the cb more generic? so that the `Database` interface doesn't need to impoirt `badger`?

Reply:
@noot All set, please take another look to confirm what I did is what you had in mind.